shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-src-distribution/pom.xml

```diff
@@ -44,6 +44,9 @@
     <profiles>
         <profile>
             <id>release</id>
+            <activation>
```

Comment:
if we make activation=true distribution will happed every time, I think we should delete it